src/magnify/registry.py

```diff
@@ -85,8 +840,221 @@ def pc_chip(**kwargs):
     return pipe
 
 
-def mrbles(**kwargs):
-    pipe = Pipeline("read", config=kwargs)
+def mrbles(
+    spectra: str | PathLike | StringIO,
+    codes: str | PathLike | StringIO,
```

Comment:
Let's move these two parameters below channels for consistency with the chip functions.

Reply:
This may be a bit hard to do as ```spectra``` and ```codes``` are non-default parameters. "channels" are default parameters as I checked and cannot come before spectra and codes.

Reply:
Good point marking as resolved. :)

Reply:
Actually, could you move data to be the first parameter? The following usage pattern is quite natural since it mimics other Python libraries such as seaborn:
mrbles(my_data, param1=a, param2=b, ...)

Reply:
Sure, I will also do this to the other function declarations in case I haven't done so